dist/privacy-sandbox.js

```diff
@@ -0,0 +1,91 @@
+//[privacy-sandbox]
+// Topics API Usage Reference: https://developers.google.com/privacy-sandbox/relevance/topics/demo#the-topics-api-demo
+// Header Usage Reference: https://developers.google.com/privacy-sandbox/relevance/topics/demo#the-topics-api-demo
+
+let requests = $WPT_BODIES;
+const testURL = new URL($WPT_URL);
+let firstPartyDomain = testURL.hostname;
+
+result = {
+  topicsAvailable: document.featurePolicy.allowsFeature('browsing-topics'),
+  thirdPartiesUsingBrowsingTopics: {}
+}
+
+function fetchAndCheckAttestation(url) {
+  const controller = new AbortController();
+  const { signal } = controller;
+  setTimeout(() => controller.abort(), 5000);
+
+  try {
+    const response = fetch(url, { signal });
+    if (response.ok && response.headers.get('Content-Type').includes('application/json')) {
+      return true;
+    }
+    else {
+      return false;
+    }
+  } catch (error) {
+    return false;
+  }
+}
+
+// Checks if the request corresponds to a JavaScript file
+function isJavaScript(requestUrl, trimmedURL, contentType) {
+  return (requestUrl.endsWith('.js') || trimmedURL.endsWith('.js') || contentType.includes('javascript'));
+}
+
+requests.forEach(request => {
+  const url = new URL(request.url);
+  const trimmedURL = url.origin + url.pathname;
+  const isScript = request.type === 'Script';
+  const isDocument = request.type === 'Document';
+
+  let thirdPartyDomain = '';
+  if (url.hostname !== firstPartyDomain) {
+    thirdPartyDomain = url.hostname;
+  }
+
+  let jsUsage = false;
+  let headerUsage = false;
+  let receiverObserving = false;
+  let attestation = false;
+  let tldPlus1Attestation = false;
+
+  // If the request is to fetch a javascript file or HTML document, perform string search in the returned file content for usage of Topics API
+  // document.browsingTopics() [JS] or fetch() request call includes: {browsingTopics: true} or XHR request call includes: {deprecatedBrowsingTopics: true}
+  if (isScript || isJavaScript(request.url, trimmedURL, request.response_headers['Content-Type']) || isDocument) {
+    if (request.response_body && (request.response_body.includes('browsingTopics') || request.response_body.includes('BrowsingTopics'))) {
+      jsUsage = true;
+    }
+  }
+
+  // Checking request header usage of Topics: header: 'Sec-Browsing-Topics: true'
+  if (request.request_headers.some(header => header.name === 'Sec-Browsing-Topics')) {
```

Comment:
```suggestion
  if ('Sec-Browsing-Topics' in request.request_headers) {
```
See object structure here: https://docs.webpagetest.org/custom-metrics/#:~:text=%22response_headers%22%3A%20%7B